Upstream-Status: Pending

automake 1.12 has deprecated use of mkdir_p, and it recommends
use of MKDIR_P instead. Changed the code to avoid these kind 
of warning-errors.

| make[2]: /build/tmp/work/x86_64-linux/libunistring-native-0.9.3-r2/image/srv/home/nitin/builds2/build0/tmp/sysroots/x86_64-linux/usr/share/doc/libunistring: Command not found
| make[2]: *** [install-html-split] Error 127

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/07/10

Index: libunistring-0.9.3/doc/Makefile.am
===================================================================
--- libunistring-0.9.3.orig/doc/Makefile.am
+++ libunistring-0.9.3/doc/Makefile.am
@@ -87,7 +87,7 @@ TEXI2DVI = @TEXI2DVI@ $(TEXINCLUDES)
 # The install-dvi target is already defined by automake.
 
 installdirs-dvi:
-	$(mkdir_p) $(DESTDIR)$(dvidir)
+	$(MKDIR_P) $(DESTDIR)$(dvidir)
 
 uninstall-dvi:
 	$(RM) $(DESTDIR)$(dvidir)/libunistring.dvi
@@ -105,7 +105,7 @@ libunistring.ps: libunistring.dvi
 # The install-ps target is already defined by automake.
 
 installdirs-ps:
-	$(mkdir_p) $(DESTDIR)$(psdir)
+	$(MKDIR_P) $(DESTDIR)$(psdir)
 
 uninstall-ps:
 	$(RM) $(DESTDIR)$(psdir)/libunistring.ps
@@ -120,7 +120,7 @@ TEXI2PDF = @TEXI2DVI@ --pdf $(TEXINCLUDE
 # The install-pdf target is already defined by automake.
 
 installdirs-pdf:
-	$(mkdir_p) $(DESTDIR)$(pdfdir)
+	$(MKDIR_P) $(DESTDIR)$(pdfdir)
 
 uninstall-pdf:
 	$(RM) $(DESTDIR)$(pdfdir)/libunistring.pdf
@@ -151,17 +151,17 @@ libunistring_toc.html: libunistring.texi
 	   }
 
 install-html-monolithic: libunistring.html
-	$(mkdir_p) $(DESTDIR)$(htmldir)
+	$(MKDIR_P) $(DESTDIR)$(htmldir)
 	$(INSTALL_DATA) `if test -f libunistring.html; then echo .; else echo $(srcdir); fi`/libunistring.html $(DESTDIR)$(htmldir)/libunistring.html
 
 install-html-split: libunistring_toc.html
-	$(mkdir_p) $(DESTDIR)$(htmldir)
+	$(MKDIR_P) $(DESTDIR)$(htmldir)
 	for file in `if test -f libunistring_toc.html; then echo .; else echo $(srcdir); fi`/libunistring_*.html; do \
 	  $(INSTALL_DATA) $$file $(DESTDIR)$(htmldir)/`basename $$file`; \
 	done
 
 installdirs-html:
-	$(mkdir_p) $(DESTDIR)$(htmldir)
+	$(MKDIR_P) $(DESTDIR)$(htmldir)
 
 uninstall-html-monolithic:
 	$(RM) $(DESTDIR)$(htmldir)/libunistring.html
@@ -170,14 +170,14 @@ uninstall-html-split:
 	$(RM) $(DESTDIR)$(htmldir)/libunistring_*.html
 
 dist-html-monolithic:
-	$(mkdir_p) $(distdir)/
+	$(MKDIR_P) $(distdir)/
 	file=libunistring.html; \
 	if test -f $$file; then d=.; else d=$(srcdir); fi; \
 	cp -p $$d/$$file $(distdir)/$$file || exit 1
 
 # We would like to put libunistring_*.html into EXTRA_DIST, but it doesn't work.
 dist-html-split: libunistring_toc.html
-	$(mkdir_p) $(distdir)/
+	$(MKDIR_P) $(distdir)/
 	file=libunistring_toc.html; \
 	if test -f $$file; then d=.; else d=$(srcdir); fi; \
 	for file in `cd $$d && echo libunistring_*.html`; do \
